tests/ssl-opt.sh

```diff
@@ -841,7 +841,7 @@ if [ "$MAX_OUT_LEN" -lt "$MAX_CONTENT_LEN" ]; then
     MAX_CONTENT_LEN="$MAX_OUT_LEN"
 fi
 
-# skip the next test if the SSL output buffer is less than 16KB
+# skip the next test if the SSL output buffer isn't equal to 16KB
```

Comment:
I'm not 100% sure but I think the right fix is to change the test a couple of lines down to `if [ "$MAX_OUT_LEN" -lt 16384 ]; then`, and leave the comment as it is.